addons/l10n_ke_edi_tremol/static/src/js/send_invoice_dialog.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module **/
+
+import { Dialog } from "@web/core/dialog/dialog";
+import { useService } from "@web/core/utils/hooks";
+
+const { Component, useState, useEffect } = owl;
+
+export class KEProxyDialog extends Component {
+    setup() {
+        this.ke_proxy = useService("l10n_ke_proxy");
+        this.action = useService("action");
+        this.state = useState(this.ke_proxy.state);
+
+        useEffect(
+            () => {
+                if (this.state.successfullySent == this.state.length &&
```

Comment:
We don't want to reload automatically unless all of them are successfully sent, `reloadRequired` is set to true if the orm is called to update the account moves, but it might only update a few of the account moves (since it might be in error), in which case we want to let the user see the error message, and then click "OK" to reload